### DISCUSSION TITLE
tail -f and miller

### DISCUSSION THREAD

**Discussion by aborruso** [johnkerl/miller#1117] [Q&A]
Hi,
I have the below input file.

Using miller 5 I can monitor on screen file change using 

```
tail -f input.csv | mlr --icsvlite --otsv --ifs "," cat
```

If I use Miller 6 I have no output on screen.

If in miller 5 I change my command in

```
mlr --icsvlite --opprint --ifs "," cat input.csv
```

I have no output.

Am I doing something wrong? Is there a more correct way to do it?
I would like to use Miller to monitor file changes, because I need to print better tail output on screen.

Thank you

```
data,stato,codice_regione
5,ITA,08
2,ITA,08
3,ITA,10
```

**Comment by johnkerl** [comment]
@aborruso sorry for the delay!!!! Somehow I missed seeing this :( 

> mlr --icsvlite --opprint --ifs "," cat input.csv

This one is as designed -- PPRINT output format needs to obtain all rows to be output before printing any, since it needs to compute maximum column widths in order to do pretty-printed alignment.

https://miller.readthedocs.io/en/latest/file-formats/#pprint-pretty-printed-tabular

The other one is a bug (and a regression ...) 😢 👀
